1-typescript-intro/sample-5-modules-1.ts

```diff
@@ -1,10 +1,28 @@
 import * as moment  from 'moment'
 import * as Chance from 'chance'
 
-var chance = new Chance();
-console.log(chance.phone());
+//var chance = new Chance();
+//console.log(chance.phone());
 
-let b = new Date("1979-05-21");
-console.log(moment(b).format("YYYY-MM-DD"))
+let b = new Date("11979-05-21");
+console.log(moment(b).format("YYYY/DD/MM"))
 
 // Zadanie: wykorzystaj moduł moment do zmodyfikowania funkcji denerującej pesel tak aby można było podać datę w formacie YYYY-DD-MM
+const GeneratePESEL = (b):string=>{
```

Comment:
the function is returning string - good, but the input 'b' should also be a string or Date